week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,18 +45,46 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+const answers = ["It is certain", "It is decidedly so", "Without a doubt", "Yes - definitely", "You may rely on it",
+"As I see it, yes","Most likely", "Outlook good", "Yes", "Signs point to yes",
+"Reply hazy, try again", "Ask again later", "Better not tell you now", "Cannot predict now", "Concentrate and ask again",
+"Don't count on it", "My reply is no", "My sources say no", "Outlook not so good", "Very doubtful"];
+
+const veryPositive = ["It is certain", "It is decidedly so", "Without a doubt", "Yes - definitely", "You may rely on it"];
+const positive = ["As I see it, yes","Most likely", "Outlook good", "Yes", "Signs point to yes"];
+const negative = ["Reply hazy, try again", "Ask again later", "Better not tell you now", "Cannot predict now", "Concentrate and ask again"];
+const veryNegative = ["Don't count on it", "My reply is no", "My sources say no", "Outlook not so good", "Very doubtful"];
+
+function shakeBall() {
+  console.log("The ball has shaken!");
+  let i = Math.floor(Math.random() * answers.length);
+  return answers[i];
+}
 
-// The answer should come from shaking the ball
-let answer;
+const result = shakeBall();
```

Comment:
How come you changed the variable name? Did you find that this affected your tests at all?